src/behaviors/ExpandToggle/ExpandToggle.js

```diff
@@ -164,7 +123,9 @@ class ExpandToggle extends React.Component {
         >
           {this.renderToggle()}
         </div>
-        {unmountClosed ? this.renderUnmountClosed() : this.renderMountClosed()}
+        <CollapseType isOpened={isExpanded} springConfig={springConfig}>
```

Comment:
should `springConfig` be renamed since we're no longer using react-spring?  Or is spring a more general term for collapsing behavior?

Reply:
Spring is a common way to describe a certain type of animation, named for basically the physics concept of springs that it follows. `react-collapse` uses some kind of spring animation in its internals, which I think is what we are configuring.